File: app/src/main/java/com/unpas/masteringmaths/student/activity/KonsepActivity.java
package com.unpas.masteringmaths.student.activity;

import android.os.Bundle;
import android.webkit.WebView;

import androidx.appcompat.app.AppCompatActivity;

import com.unpas.masteringmaths.R;

import static com.unpas.masteringmaths.utils.UtilsConstant.NUMBER_INDEX;

public class KonsepActivity extends AppCompatActivity {

    private WebView konsep;
    private int index;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_student_konsep);
        konsep = findViewById(R.id.main_konsep);
        konsep.getSettings().setBuiltInZoomControls(true);

        index = getIntent().getIntExtra(NUMBER_INDEX, 0);

        switch (index) {
            case 0:
                konsep.loadUrl("file:///android_asset/petakonsepkartesius.html");
                break;
            case 1:
                konsep.loadUrl("file:///android_asset/petakonsepgarislurus.html");
                break;
            case 2:
                konsep.loadUrl("file:///android_asset/petakonseppolabilangan.html");
                break;
            case 3:
                konsep.loadUrl("file:///android_asset/petakonseppythagoras.html");
                break;
            case 4:
                konsep.loadUrl("file:///android_asset/petakonsepspldv.html");
                break;
        }
    }
}

File: app/src/main/java/com/unpas/masteringmaths/student/activity/KonsepActivity.kt
package com.unpas.masteringmaths.student.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.unpas.masteringmaths.R
import com.unpas.masteringmaths.utils.UtilsConstant
import kotlinx.android.synthetic.main.activity_student_kompetesi.*
import kotlinx.android.synthetic.main.toolbar_layout.*

class KonsepActivity : AppCompatActivity() {

    private var index = 0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_student_kompetesi)
        setSupportActionBar(toolbar)
        supportActionBar?.title = "Peta Konsep"
        main_content.settings.builtInZoomControls = true
        index = intent.getIntExtra(UtilsConstant.NUMBER_INDEX, 0)

        when (index) {
            0 -> main_content.loadUrl("file:///android_asset/petakonsepkartesius.html")
            1 -> main_content.loadUrl("file:///android_asset/petakonsepgarislurus.html")
            2 -> main_content.loadUrl("file:///android_asset/petakonseppolabilangan.html")
            3 -> main_content.loadUrl("file:///android_asset/petakonseppythagoras.html")
            4 -> main_content.loadUrl("file:///android_asset/petakonsepspldv.html")
        }
    }
}